### DISCUSSION TITLE
nuclei templates missing

### DISCUSSION THREAD

**Discussion by negritocucu** [projectdiscovery/nuclei#5438] [Q&A]
Hello, I have problems with nuclei, I install nuclei with go, and it also installs the nuclei-templates automatically, but it does not give me all the cves, there is only one. What is the problem, excuse me, please.

![kali1](https://github.com/user-attachments/assets/d2c7ce05-eb54-48ef-8cbf-5da63fad1091)
![kali2](https://github.com/user-attachments/assets/c63dd688-8d22-4b02-a3dd-e882b35adac2)

**Comment by ehsandeep** [comment]
@negritocucu nuclei templates directory is structured based on protocol and since of CVEs are based on HTTP protocol, all those CVEs templates you are looking for goes into `http/cves` directory.
